spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringDocSwaggerConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.springdoc.demo.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+public class SpringDocSwaggerConfig {
+
+	@Configuration
+	@EnableSwagger2
+	public class SwaggerConfig {
```

Comment:
When declaring an inner class, it's generally recommended to make it static if it doesn't need to access instance-specific data or methods of the outer class. So could you

Reply:
removed